### PR TITLE
Update DotNetty versions

### PR DESCRIPTION
**Pull request by schoims**

### CODE REVIEW

File: iothub/device/src/Microsoft.Azure.Devices.Client.csproj [hunk 1]
```diff
@@ -68,10 +68,10 @@
     <Compile Include="ModernDotNet\HsmAuthentication\Transport\HttpBufferedStream.cs" />
     <Compile Include="ModernDotNet\HsmAuthentication\Transport\HttpRequestResponseSerializer.cs" />
     <Compile Include="ModernDotNet\HsmAuthentication\Transport\UnixDomainSocketEndPoint.cs" />
-    <PackageReference Include="DotNetty.Codecs.Mqtt" Version="0.7.0" />
-    <!-- We will need to keep this reference to DotNetty.Common until the next release of DotNetty includes this dependency -->
-    <PackageReference Include="DotNetty.Common" Version="0.7.1" />
-    <PackageReference Include="DotNetty.Handlers" Version="0.7.0" />
+    <PackageReference Include="DotNetty.Codecs" Version="0.7.5" />
```
Comment (abhipsaMisra) [inline iothub/device/src/Microsoft.Azure.Devices.Client.csproj:71]: Is this a new package that we need to take a dependency on now?

File: iothub/device/src/Microsoft.Azure.Devices.Client.csproj [hunk 2]
```diff
@@ -68,10 +68,10 @@
...
+    <PackageReference Include="DotNetty.Codecs.Mqtt" Version="0.7.5" />
+    <PackageReference Include="DotNetty.Common" Version="0.7.5" />
```
Comment (abhipsaMisra) [inline iothub/device/src/Microsoft.Azure.Devices.Client.csproj:73]: DotNetty.Common should be referenced by either DotNetty.Codecs.Mqtt or DotNetty.Handlers, so we should be ok removing this dependency.

File: iothub/device/tests/Microsoft.Azure.Devices.Client.Tests.csproj [hunk 3]
```diff
@@ -23,10 +23,6 @@
     <Compile Include="HsmAuthentication\**" />
     <Compile Remove="ClientWebSocketTransportTests.cs" />
     <Compile Remove="Mqtt\ClientWebSocketChannelTests.cs" />
-    <PackageReference Include="DotNetty.Codecs.Mqtt" Version="0.7.0" />
-    <!-- We will need to keep this reference to DotNetty.Common until the next release of DotNetty includes this dependency -->
-    <PackageReference Include="DotNetty.Common" Version="0.7.1" />
-    <PackageReference Include="DotNetty.Handlers" Version="0.7.0" />
```
Comment (schoims) [inline iothub/device/tests/Microsoft.Azure.Devices.Client.Tests.csproj:29]: removing unused dependencies

**Review by abhipsaMisra — APPROVED** [review]

**Review by drwill-ms — APPROVED** [review]

**Pull request merged**